### DISCUSSION TITLE
Bootstrap vs Tailwind CSS

### DISCUSSION THREAD

**Discussion by martinszeltins** [tailwindlabs/tailwindcss#8032] [General]
What are the benefits of using Tailwind CSS vs Bootstrap? You can customize Bootstrap to make it look whatever you like and you can add components to Bootstrap and make your own custom Bootstrap theme. So what benefit does Tailwind have?

**Comment by BhavikTrambadiya** [comment]
## The main benefits are that 

1. More flexible to developer website without leaving the HTML.
2. Structure your CSS when teammates work on the same project.
3. Improve your theme according to all possible ways with a simple setup file
4. Reduce the CSS file size
5. Remove the media query pain from the project

### Above all is the just simple benefit but if you are interested in the tailwind that read the document and tries to make a small project I am sure tailwind makes you addicted to using it.

**Comment by junctionapps** [comment]
Hi @martinszeltins I've been toying with Tailwind CSS for a little bit. For me it gives a lot of ability to do things I couldn't figure out how to do in Bootstrap. I'm sure you can in Bootstrap (I used it for years), but Tailwind CSS seemed like an easier/natural fit for me. Some of the stuff over at https://www.youtube.com/c/TailwindLabs really highlights some of the strengths. For me it gives a more enjoyable experience.
